zarr/hierarchy.py

```diff
@@ -414,6 +416,78 @@ def arrays(self):
                                  chunk_store=self._chunk_store,
                                  synchronizer=self._synchronizer)
 
+    def visit(self, func):
+        """Run callable on each object name.
+
+        Examples
+        --------
+        >>> from __future__ import print_function, unicode_literals
```

Comment:
I don't think this is needed in the example. For Zarr docstrings are standardised to PY3(6).